### PR TITLE
Morgue holosign

### PR DESCRIPTION
**Pull request by DamianX**
It looks like this:
image
I codersprited it in five minutes by copying the surgery holosign, so if someone wants to make a better one they are very welcome.

It lights up if any morgue tray has a green light.

~~I will add it to all maps if this PR goes through.
No changelog because this PR doesn't actually add it to any map~~

:cl:
* rscadd: Added a morgue holosign to all morgue airlocks. They light up if there's one or more revivable bodies in the morgue trays.

### CODE REVIEW

**Comment by toomykins** [comment]
those morgues should be made hackable/emaggable tbh so they dont report life.

**Comment by tuypo1** [comment]
Nah too easy Toomy, you should be expected to actually dispose properly of bodies

**Comment by Blithering** [comment]
So does it connect to all morgue trays or just those inside the morgue?

**Comment by DamianX** [comment]
It connects to all morgue trays inside the area. So if you put one of these in the ghetto morgue and one in the real morgue, they will light up when the respective morgue trays are filled.

File: code/game/machinery/holosign.dm [hunk 1]
```diff
@@ -67,6 +67,28 @@ var/list/obj/machinery/holosign/holosigns = list()
...
+	id_tag = "morgue"
+	var/should_update = FALSE
+
+/obj/machinery/holosign/morgue/process()
+	var/area/this_area = get_area(src)
+	for(var/obj/structure/morgue in morgue_list)
+		var/area/morgue_area = get_area(morgue)
+		if(morgue_area != this_area)
+			continue
+		var/morgue_has_revivable_dude = morgue.icon_state == "morgue4"
```
Comment (ShiftyRail) [inline code/game/machinery/holosign.dm]: The lack of parenthesis to this statement upsets me

**Comment by ShiftyRail** [comment]
Does the holosign automatically switch off after I remove the revivable body, or do I need to turn it off myself ?
Does it turn off after the body becomes unrevivable (ie, if someone ghosts after waiting too long in the morgue) ?

**Comment by DamianX** [comment]
It turns off automatically when there are no revivable corpses.
It does turn off when someone ghosts out.

File: code/game/machinery/holosign.dm (hunk 1 above)
Comment (DamianX) [reply code/game/machinery/holosign.dm]:
```suggestion
		var/morgue_has_revivable_dude = (morgue.icon_state == "morgue4")
```

**Comment by bathosbathtime** [comment]
Does it draw from APC power, and from what type? If it draws from the light subsection of the morgue that could be a way to hack it off.
Not that this is a priority but it would be nice to have a way of working around it.

**Comment by DamianX** [comment]
That was a good question. As it turns out, holosigns were supposed to turn off when the room's equipment was off, but that never worked.
So now this PR unatomically makes holosigns:
- turn off in the absence of power
- use the "lights" power channel instead of "equipment"

**Comment by DamianX** [comment]
Since it is taking forever for this to get merged I went ahead and added the holosigns to all maps.

**Pull request merged**